tests/unittests/Foundation/NSHttpCookieTests.mm

```diff
@@ -17,18 +17,19 @@
 #include <TestFramework.h>
 #import <Foundation/Foundation.h>
 
-// Implemented in NSHTTPCookie.mm
-extern void parseCookies(const char* lineptr, id dict);
+// // Implemented in NSHTTPCookie.mm
```

Comment:
Oops, probably put this away for a sec for whatever reason, then forgot to revert.

---
In reply to: [72314834](https://github.com/Microsoft/WinObjC/pull/634#discussion_r72314834) [](ancestors = 72314834)